README.md

```diff
@@ -4,6 +4,50 @@ My solutions to [Advent of Code 2023].
 
 [Advent of Code 2023]: https://adventofcode.com/2023
 
+## To run
+Use :
+```lein run -m t.day01```
+in the terminal.
+
+Change `day01` as needed.
+
+## Lessons (learned?)
+
+### Day 01
+
+Using `Integer.` is a bad idea.\
+Capital Letter ==> java class\
+Capital Letter + "." ==> constructor of the java class :\
+    - creates new object
+    - Integer : in 32bits  ==> max is 2.10^9 : not that much with advent of code and no warning/error
+Solution : better to use integer 64bit: Long\
+Let's **not** use `Long.` because again new object... instead : **`parse-long`**
+
+
+### Day 02
+
+#### `reduce` and `and`
+
+```clojure
+(defn iscolorok?
+  [color numb cubes]
+  (reduce (fn [a b] (and a b)) true
+          (map (fn [b] (<= (parse-long (re-find #"\d+" b)) numb))
+               (re-seq (re-pattern (str "\\d+ " color)) cubes))))
+```
+`reduce` needs a function of two elements\
+`and` is _not_ a function so we have to define one\
+we set `true` as a starting point in case the sequence has only one element
```

Comment:
It's also worth taking a look at [`every?`](https://clojuredocs.org/clojure.core/every_q), which can be defined as `(every? f coll)` being semantically equivalent to `(reduce #(and %1 %2) true (map f coll))`.